aws/scenarios/microVMs/vmconfig/load.go

```diff
@@ -0,0 +1,50 @@
+package vmconfig
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"regexp"
+)
+
+func LoadFile(filename string) (*Config, error) {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Since this file is supposed to be a configuration file, would it make sense to call this function `LoadConfigFile` or `ParseFromFile` ?

Note you could use `embed` here too and `ParseFromString`